examples/federated-connections-with-access-tokens/chatbot/SDK_NOTES.md

```diff
@@ -0,0 +1,104 @@
+# Enhanced Federated Connections with Access Tokens
```

Comment:
Preliminary SDK notes based on my prompts while implementing a preliminary SDK change today. We can certainly remove these or incorporate into the main README later as needed.